src/archive.js

```diff
@@ -1,9 +1,16 @@
 /**
  * @fileoverview Keeps track of prior action states according to an
  * action's id
+ * @flow
  */
 
+import type Action from './action'
+
+type Payload = { [string]: * }
```

Comment:
Yeah. It is an object where the keys are strings and the values could be anything.